static/app/views/replays/replayTable/index.tsx

```diff
@@ -140,13 +190,21 @@ function ReplayTable({
   );
 }
 
+const replayDetailCells = [
+  ReplayColumn.REPLAY,
+  ReplayColumn.MOST_DEAD_CLICKS,
+  ReplayColumn.MOST_ERRONEOUS_REPLAYS,
+];
```

Comment:
it's a nit, but the variable name could be like: `flexibleColumns` or something like that.

if line 206 was a proper english sentence it would be something like "if the $column is one of the $flexibleColumns then print 'minmax'. By default print 'max-content'"

```suggestion
const replayDetailCells = [
  ReplayColumn.REPLAY,
  ReplayColumn.MOST_DEAD_CLICKS,
  ReplayColumn.MOST_ERRONEOUS_REPLAYS,
];
```